src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier;
+    private final Random random;
+    private final int maxRandomNumber;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
+        this.random = new Random();
+        this.maxRandomNumber = 100;
```

Comment:
Constants are written in the upper case and static final. Also initialized in the declaration line